src/app/@ansyn/ansyn/app-effects/app.effects.module.ts

```diff
@@ -35,6 +36,7 @@ export interface IAppState {
 @NgModule({
 	imports: [
 		TranslateModule,
+		StoreModule.forFeature(casesFeatureKey, CasesReducer),
```

Comment:
why not in cases module?

Reply:
after recheck it's seem like it's unnecessary import